package.json

```diff
@@ -3,127 +3,202 @@
     "babel-preset-nano-react-app": "^0.1.0",
+    "core-js": "^3.11.1",
     "css-loader": "^3.4.2",
-    "eslint": "^6.6.0",
-    "eslint-config-prettier": "^6.5.0",
-    "eslint-config-standard": "^14.1.0",
-    "eslint-plugin-import": "^2.18.2",
-    "eslint-plugin-node": "^10.0.0",
-    "eslint-plugin-prettier": "^3.1.1",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-react": "^7.16.0",
-    "eslint-plugin-standard": "^4.0.1",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.6",
+    "eslint": "^7.32.0",
+    "eslint-import-resolver-typescript": "^2.0.0",
+    "eslint-import-resolver-webpack": "^0.12.1",
+    "eslint-plugin-header": "^3.1.1",
+    "eslint-plugin-import": "^2.25.3",
+    "eslint-plugin-mdx": "^1.16.0",
+    "eslint-plugin-prettier": "^4.0.0",
     "file-loader": "^6.0.0",
-    "identity-obj-proxy": "^3.0.0",
-    "jest": "^25.2.4",
-    "prettier": "^1.19.1",
-    "react-test-renderer": "^16.13.1",
+    "jest": "26.6.3",
+    "jest-styled-components": "^7.0.3",
+    "prettier": "2.2.1",
+    "react-hot-loader": "^4.12.20",
+    "react-test-renderer": "^17.0.1",
     "style-loader": "^1.1.3",
+    "ts-jest": "^26.5.5",
     "ts-loader": "^6.2.2",
-    "typescript": "^3.8.3"
+    "typescript": "^4.2.4",
+    "webpack": "^5.10.0",
+    "webpack-bundle-analyzer": "^4.4.1",
+    "webpack-cli": "^4.6.0",
+    "webpack-dev-server": "^3.11.2"
   },
-  "babel": {
-    "presets": [
-      "@babel/preset-react",
-      "@babel/preset-typescript",
-      [
-        "@babel/preset-env",
-        {
-          "targets": {
-            "esmodules": true
-          }
-        }
-      ]
+  "eslintConfig": {
+    "extends": [
+      "@looker/eslint-config-oss"
     ],
-    "plugins": [
-      [
-        "@babel/plugin-proposal-class-properties",
+    "rules": {
+      "@typescript-eslint/no-explicit-any": "off",
+      "camelcase": "off",
+      "@typescript-eslint/interface-name-prefix": "off",
+      "@typescript-eslint/no-unused-vars": [
+        "warn",
         {
-          "loose": true
+          "args": "all",
+          "argsIgnorePattern": "^_"
         }
       ],
-      [
-        "@babel/plugin-transform-react-jsx",
+      "sort-keys-fix/sort-keys-fix": "off",
+      "no-useless-constructor": "off",
+      "@typescript-eslint/no-empty-interface": "off",
+      "import/default": "off",
+      "sort-keys": "off",
+      "spaced-comment": [
+        "error",
+        "always",
         {
-          "pragmaFrag": "React.Fragment"
+          "markers": [
+            "#region",
+            "#endregion"
+          ]
         }
+      ],
+      "no-use-before-define": "off",
+      "no-console": 0,
+      "header/header": [
+        2,
+        "block",
+        [
+          "",
+          "",
+          " MIT License",
+          "",
+          {
+            "pattern": " Copyright \\(c\\) 20\\d{2} Looker Data Sciences, Inc.",
+            "template": " Copyright (c) 2022 Looker Data Sciences, Inc."
```

Comment:
Should this be 2023 now?

Reply:
Yes. Fixed but I'm not going to fix the files that have already been updated to 2022.